test/core/Safe.GuardManager.spec.ts

```diff
@@ -25,14 +24,21 @@ describe("GuardManager", () => {
         const [, user2] = signers;
 
         const guardContract = await hre.ethers.getContractAt("Guard", AddressZero);
-        const guardEip165Calldata = guardContract.interface.encodeFunctionData("supportsInterface", ["0x945b8148"]);
+        const guardEip165Calldata = guardContract.interface.encodeFunctionData("supportsInterface", ["0xe6d7a83a"]);
         await validGuardMock.givenCalldataReturnBool(guardEip165Calldata, true);
         const safe = await getSafeWithOwners([user2.address]);
         await executeContractCallWithSigners(safe, safe, "setGuard", [validGuardMockAddress], [user2]);
 
+        const validModuleGuardMock = await getMock();
+        const moduleGuardContract = await hre.ethers.getContractAt("IModuleGuard", AddressZero);
+        const moduleGuardEip165Calldata = moduleGuardContract.interface.encodeFunctionData("supportsInterface", ["0xd7e8e3a4"]);
+        await validModuleGuardMock.givenCalldataReturnBool(moduleGuardEip165Calldata, true);
+        await executeContractCallWithSigners(safe, safe, "setModuleGuard", [validModuleGuardMock.target], [user2]);
+
         return {
             safe,
             validGuardMock,
+            validModuleGuardMock,
```

Comment:
This mock could be removed, and subsequent setup as well.

Reply:
aah...this was left out as I moved some tests to `Safe.ModuleManager.spec.ts`

Reply:
https://github.com/safe-global/safe-smart-account/pull/758/commits/924bc59ae48edb360aeaac33bc78a6d85ce17c46